mqtt_as/mqtt_as.py

```diff
@@ -382,7 +403,9 @@ def close(self):
         while not self.suback:
```

Comment:
All subscribe listen to self.suback so they all return "successfully" once one subscribe receives a SUBACK, even if wait_msg raises an OSError for the next one or the connection breaks.